src/includes/configuration/session-tracking-interval/android.mdx

```diff
@@ -3,3 +3,14 @@
   <meta-data android:name="io.sentry.session-tracking.timeout-interval-millis" android:value="30000" />
 </application>
 ```
+
+Or, if you are manually instrumenting Sentry:
+
+```java
```

Comment:
`l`: Not really related to this PR, but should we maybe also provide all code samples in Kotlin?

Reply:
it'd be nice, but not sure if it's worth it, nowadays the IDE always translate Java -> Kotlin automatically anyway when ctrl c ctrl v, maybe something for the future

Reply:
@maciejwalkowiak is adding both Java and Kotlin.
Would be really nice to have Kotlin for Android too. You can add them both on the same file, one on top of the other

Reply:
mmm true, not true, the only snippet that has both languages is configurations -> https://docs.sentry.io/platforms/java/usage/#configuration

I'm all in for doing it, but I'd do it in the next PR when all the code snippets are stable/reviewed as we'd need to convert them if that's ok.

Reply:
I did not want to add Kotlin equivalents until we are confident that Java samples are good.

Once we have Java all Java samples merged I will add Kotlin equivalent.